src/actions/commentActions.js

```diff
@@ -0,0 +1,49 @@
+import commentAPI from '../api/mockCommentAPI';
+import * as types from './actionTypes';
+
+export function loadCommentsRequest() {
```

Comment:
Make sure to run `npm run lint` over all of this. The config I'm using might make you use `const loadCommentsRequest = {};` instead of full functions here.